spec/models/client_spec.rb

```diff
@@ -49,7 +49,9 @@ def app
     client = Client.create(name: 'Delete Me')
     expect(client).to exist
     expect { client.delete }.to_not raise_error
+    ### Test passes but the item's name and address can still be printed?
     expect(client).to_not exist
+    print client, ' ', client.name
```

Comment:
Yes, I think that is correct, Model.delete removes the database entry and returns self (the object you are trying to delete). Probably the way to test this is to try and find the object in the database again and test that it isn't there anymore - or, use Sequel's model method 'exists?' which tests if the object exists in the database (expect client.exists? to eq false)